src/plugins/blocksmith/components/sidebar/config.js

```diff
@@ -1,8 +1,9 @@
+/* globals APP_VENDOR */
 export default {
   enableOther: true,
   groups: [
     {
-      header: 'Triniti',
+      header: APP_VENDOR,
```

Comment:
@rsumilang need to use the localizationmap here

Reply:
Localization map can be updated on a per vendor app basis.